schemas/apps_meta.schema.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
What is the use of this file? `apps_meta.json` is generated on the aiidalab server, not in here. Is it added here for the reference?

Reply:
The `apps_meta.json` file effectively represents an endpoint that some of our applications and services are users of. It is not added for reference, but to have a well-defined standard of what this endpoint returns.

Reply:
The `apps_meta.json` file effectively represents an endpoint that some of our applications and services are users of. It is not added for reference, but to have a well-defined standard of what this endpoint returns.